src/Storage/Storage.Management.Sdk/Generated/Models/RestorePolicyProperties.cs

```diff
@@ -86,16 +86,13 @@ public RestorePolicyProperties(bool enabled, int? days = default(int?), System.D
         /// </exception>
         public virtual void Validate()
         {
-            if (Days != null)
```

Comment:
The code is not generated with the correct version of Autorest.
Please run following command to update autorest, and then re-generate it:

```
autorest --reset
autorest --use:@microsoft.azure/autorest.csharp@2.3.90
```